webapp/packages/plugin-codemirror6/src/ReactCodemirror.tsx

```diff
@@ -108,7 +105,7 @@ export const ReactCodemirror = forwardRef<IEditorRef, IReactCodeMirrorProps>(fun
     if (view) {
       view.dispatch({ effects: StateEffect.reconfigure.of(ext) });
     }
-  }, [extensions, readonly, editable, onChange, onUpdate]);
+  }, [extensions, view]);
```

Comment:
you can just skip dependencies, because it's will be executed on every re-render

Reply:
also probably you can call reconfigure on EditorState instead